src/components/include/cos_kernel_api.h

```diff
@@ -88,6 +89,8 @@ int cos_sched_rcv(arcvcap_t rcv, thdid_t *thdid, int *blocked, cycles_t *cycles)
 
 int cos_introspect(struct cos_compinfo *ci, capid_t cap, unsigned long op);
 
+int cos_sinv(sinvcap_t sinv);
```

Comment:
crap this is the last thing not changed in that latest push. I didn't realize @RobertGiff had updated his sinv() implementation since I last merged. will port that now.